scripts/tfsemb_LMBase.py

```diff
@@ -49,21 +50,22 @@ def main():
     args = arg_parser()
     setup_environ(args)
 
-    base_df = load_pickle(args.labels_pickle, "labels")
+    # base_df = load_pickle(args.labels_pickle, "labels")
+    base_df = pd.read_pickle(args.labels_pickle)
 
     glove = api.load("glove-wiki-gigaword-50")
     base_df["in_glove50"] = base_df.word.str.lower().apply(
         lambda x: isinstance(glove.key_to_index.get(x), int)
     )
 
     if args.embedding_type == "glove50":
-        base_df = base_df[base_df["in_glove50"]]
+        # base_df = base_df[base_df["in_glove50"]]  # turning off to ensure glove base_df has same shape as datum
         base_df = add_vocab_columns(args, base_df, column="word")
     else:
         base_df = tokenize_and_explode(args, base_df)
-        # base_df = add_vocab_columns(args, base_df, column="token2word")
+        # base_df = add_vocab_columns(args, base_df, column="token2word")  #FIXME: should we include this here?
```

Comment:
It might be helpful but probably not needed for now. I turned it off because it takes a long time and adds a lot of columns that we don't need. Ideally, if we can merge every model token to words, then we don't need to align tokens between models

Reply:
I agree. It can be left out as we should be able to merge the base_df and the embedding_df now with a simply `group_by` on the latter.